packages/backend/src/graphql/schema.graphql

```diff
@@ -1204,116 +1192,64 @@ interface Model {
   createdAt: String!
 }
 
-type CreateWalletAddressMutationResponse implements MutationResponse {
-  code: String!
-  success: Boolean!
-  message: String!
+type CreateWalletAddressMutationResponse {
   walletAddress: WalletAddress
 }
 
-type UpdateWalletAddressMutationResponse implements MutationResponse {
-  code: String!
-  success: Boolean!
-  message: String!
+type UpdateWalletAddressMutationResponse {
   walletAddress: WalletAddress
 }
 
-type TriggerWalletAddressEventsMutationResponse implements MutationResponse {
-  code: String!
-  success: Boolean!
-  message: String!
+type TriggerWalletAddressEventsMutationResponse {
   "Number of events triggered"
   count: Int
 }
 
-type AssetMutationResponse implements MutationResponse {
-  code: String!
-  success: Boolean!
-  message: String!
+type AssetMutationResponse {
   asset: Asset
 }
 
-type DeleteAssetMutationResponse implements MutationResponse {
-  code: String!
-  success: Boolean!
-  message: String!
+type DeleteAssetMutationResponse {
+  asset: Asset
 }
 
-type CreatePeerMutationResponse implements MutationResponse {
-  code: String!
-  success: Boolean!
-  message: String!
+type CreatePeerMutationResponse {
   peer: Peer
 }
 
-type CreateOrUpdatePeerByUrlMutationResponse implements MutationResponse {
-  code: String!
-  success: Boolean!
-  message: String!
+type CreateOrUpdatePeerByUrlMutationResponse {
   peer: Peer
 }
 
-type UpdatePeerMutationResponse implements MutationResponse {
-  code: String!
-  success: Boolean!
-  message: String!
+type UpdatePeerMutationResponse {
   peer: Peer
 }
 
-type DeletePeerMutationResponse implements MutationResponse {
-  code: String!
-  success: Boolean!
-  message: String!
-}
-
-type TransferMutationResponse implements MutationResponse {
-  code: String!
+type DeletePeerMutationResponse {
   success: Boolean!
-  message: String!
 }
 
-type LiquidityMutationResponse implements MutationResponse {
-  code: String!
-  success: Boolean!
-  message: String!
-  error: LiquidityError
+type LiquidityMutationResponse {
+  id: ID!
+  liquidity: UInt64
```

Comment:
Actually I see we are already doing `success: Boolean!` in `DeletePeerMutationResponse`. Perhaps that would be appropriate here as well? 

The caller already has the returned `id` and `liquidity` values anyways, since what we are returning is just coming from the input.

Reply:
I used this approach as a holdover from when I took this over from [Sabine's branch on this](https://github.com/interledger/rafiki/pull/2667/files#diff-fdaa49a1159ec466e5ccb9daa6e086e9188bb521d630840a15f67f1188c9443c). I think returning a `success` boolean makes sense but I'm curious to hear if @sabineschaller has any particular reasoning for this before switching it over.

Reply:
Based on today's call it doesn't seem like she's all that attached to whatever approaches she used before I took over. I also think the `success` boolean makes more sense so I'll change it to use that as well.